enclave-tls/src/attesters/internal/etls_enclave_attester_load_single.c

```diff
@@ -6,14 +6,17 @@
 
 #include <stdlib.h>
 #include <string.h>
-#include <dlfcn.h>
 #include <enclave-tls/err.h>
 #include <enclave-tls/log.h>
 #include "internal/core.h"
 #include "internal/attester.h"
 
-#define PATTERN_PREFIX "libattester_"
-#define PATTERN_SUFFIX ".so"
+#define PATTERN_PREFIX  "libattester_"
+#ifdef SGX
+#define PATTERN_SUFFIX  ".a"
```

Comment:
.a is not runtime component. You cannot assume a running system deploys .a files to satisfy this logic.